modules/ROOT/pages/a11ychecker.adoc

```diff
@@ -29,7 +29,7 @@ For example:
 tinymce.init({
   selector: 'textarea',
   plugins: 'a11ychecker',
-  toolbar: 'a11ycheck'
+  toolbar: 'a11ycheck',
```

Comment:
this is not necessary

Reply:
I agree with @ArvinJ-H even though this might make it easier to copy and paste, we should remove it.